make/kube

```diff
@@ -6,17 +6,22 @@ GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
 cd "${GIT_ROOT}"
 source .envrc
 
-ENV_FILES="$(echo env/*.env)"
-
-CREATE_HELM_CHART=false
-BUILD_TARGET=kube
+DOMAIN=${DOMAIN:-}
+if test -n "$DOMAIN"; then
+    # If DOMAIN is set, place DOMAINT at the end of defaults.env temporarily
+    echo "DOMAIN=$DOMAIN" >> env/defaults.env
+    trap "sed -i '$d' env/defaults.env" EXIT
```

Comment:
/home/vagrant/scf/src/uaa-fissile-release/make/kube: line 13: d: unbound variable

Yes. The outer `"`-quoting causes interpolation of `$d`, `d` does not exist.
Has to be `\`-escaped.